<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFindReplaceWidget</class>
 <widget class="QWidget" name="CFindReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find and Replace Text</string>
  </property>
  <widget class="QTextEdit" name="m_pTextEditLookFor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>780</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditInputOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>780</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelReplaceWith">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace with:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditReplaceWith">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>780</width>
     <height>90</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnFindReplaceText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>FIND AND REPLACE TEXT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckBoxGlobalMatching">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Global matching.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckBoxCaseSensitive">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Case sensitive.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSelAllOutputText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnClearAllOutputText">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnLoadFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOAD FILE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnBrowse">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnDos">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>530</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Dos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSaveAs">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE AS</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnUnix">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>530</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Unix</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditSaveAs">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelWrapIs">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wrap is</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnWrapOn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>530</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>on</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnWrapOff">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>530</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>off</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pFindThisLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Find this:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckBoxRegularExp">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable regular expression</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSelectedFile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelResult">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
